src/neptune/new/internal/disk_queue.py

```diff
@@ -50,15 +50,15 @@ def __init__(
         from_dict: Callable[[dict], T],
         lock: threading.RLock,
         max_file_size: int = 64 * 1024**2,
-        max_batch_size: int = None,
+        max_batch_size_bytes: int = None,
     ):
         self._dir_path = dir_path.resolve()
         self._to_dict = to_dict
         self._from_dict = from_dict
         self._max_file_size = max_file_size
-        self._max_batch_size = max_batch_size
-        if max_batch_size is None:
-            self._max_batch_size = int(os.environ.get("MAX_BATCH_SIZE") or str(100 * 1024**2))
+        self._max_batch_size_bytes = max_batch_size_bytes
+        if max_batch_size_bytes is None:
+            self._max_batch_size_bytes = int(os.environ.get("NEPTUNE_MAX_BATCH_SIZE_BYTES") or str(100 * 1024**2))
```

Comment:
Even shorter:

```
self._max_batch_size_bytes = max_batch_size_bytes or
    int(os.environ.get("NEPTUNE_MAX_BATCH_SIZE_BYTES") or str(100 * 1024**2))
```

Reply:
And let's declare some constant inside this class instead of using maginc number 

`DEFAULT_MAX_BATCH_SIZE_BYTES = 100 * 1024**2`.